core/src/avm2/globals/flash/display3D/context_3d.rs

```diff
@@ -298,3 +369,117 @@ pub fn clear<'gc>(
     }
     Ok(Value::Undefined)
 }
+
+pub fn create_texture<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    if let Some(context) = this.and_then(|this| this.as_context_3d()) {
+        // This is a native method, so all of the arguments have been checked and coerced for us
+        let width = args[0].as_integer(activation.context.gc_context)? as u32;
```

Comment:
future nit: `as_integer` is still relatively heavyweight (calls `value_of` + error text formatting), so might wanna smth even simpler if the arguments were checked